extensions/interactions/LogicProof/directives/logic-proof-interaction.directive.html

```diff
@@ -1,9 +1,10 @@
 <div class="logic-proof-container">
   <span translate="<[$ctrl.questionString]>" translate-values="$ctrl.questionStringData"></span>
   <div class="logic-proof-input-box">
-    <div ui-codemirror="{onLoad: $ctrl.codeEditor, readOnly: !$ctrl.interactionIsActive}">
-      <textarea ng-trim="false" rows="10"></textarea>
-    </div>
+    <oppia-codemirror (on-load)="$ctrl.codeEditor($event)"
+                      [read-only]="!$ctrl.interactionIsActive">
+    </oppia-codemirror>
+    <textarea ng-trim="false" rows="10"></textarea>
```

Comment:
are you sure about textarea tag here?

https://user-images.githubusercontent.com/24855641/117271316-ace59380-ae77-11eb-930d-d9d67ba2765e.mp4

Before this PR, there was no extra textarea except the codemirror.
this is causing additionalPlayerFeatures suite to fail.
Removing the textarea tag seems to fix the issue. If this is unexpected, can i fix this in my learner-local-nav PR?

Reply:
The tag has to be removed. Please fix it in that PR.

Reply:
done